index.html

```diff
@@ -0,0 +1,535 @@
+            </div>
+        </div>
+    </div>
+</div>
+
+</section>
+
+    <section class="contact-form">
+        <div class="form-text">
+            <h2 class="text-section">I'm always <br> interested in <br> hearing about <br> new projects, so <br> if you'd like to <br> chat please get <br> in touch.</h2>
+        </div>
+            <form action="https://formspree.io/f/myyvzkbz" method="post">
+                <label for="name">
+                    <input type="text" name="name" id="name" placeholder="Full name" maxlength="30" required>
+                </label>
+                <label for="email">
+                  <input type="email" name="email" id="email" class="input-cls" required
+                  placeholder="Email address" pattern="[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Za-z]{1,63}$" />
+                </label>
+                <label for="message">
+                  <textarea class="input-cls" name="message" id="message" cols="48" rows="9" maxlength="500"
+                  placeholder="Enter text here" required></textarea>
+                </label>
+                <button type="submit" class="btn-green">Get in touch</button>
+            </form>
+
+        <ul class="social-media-icon">
+            <li><a href="https://twitter.com/siddharthaghos9"><img src="./images/NormalButton2/Tertiary/Icons/twitter.png" class="twitter" alt="Twitter icon"></a></li>
+            <li><a href="https://www.linkedin.com/in/siddhartha-ghosh-65902718/"><img src="./images/NormalButton2/Tertiary/Icons/linkedin.png" class="linkedin" alt="Linkedin icon"></a></li>
+            <li><a href="#"><img src="./images/NormalButton2/Tertiary/Icons/Vector.png" class="vector" alt="Vector icon"></a></li>
+            <li><a href="https://github.com/siddghosh108"><img src="./images/NormalButton2/Tertiary/Icons/github.png" class="github" alt="Github icon"></a></li>
+            <li><a href="#"><img src="./images/NormalButton2/Tertiary/Icons/angelis.png" class="angelis" alt="Angelis icon"></a></li>
+        </ul>
+
+        <div class="contact-bg-image">
+        </div>
+
+    </section>
+    
+        <div class="email-text">
+            <p class="supporting-text-2">
+                sikonorbert0@gmail.com
+            </p>
+        </div>
+
+    
+<!-- END OF MOBILE VERSION-->
+
+
+    <!-- START OF DESKTOP VERSION -->
+
+    <div class="desktop-background-images">
+    </div>
+
+    <!-- Desktop version header -->
+    <header class="header">
+        <ul class="menu">
+            <li class="menu menu-items1"><a href="#"></a>Portfolio</li>
+            <li class="menu menu-items2"><a href="#"></a>About</li>
+            <li class="menu menu-items3"><a href="#"></a>Contact</li>
+        </ul>
+    </header>
+    
+    <!-- Desktop version headline -->
+    <section class="desktop-headline">
+        <div class="desktop-headline-first-text">
+            <h2 class="desktop-headline-heading">Hey there. I’m Siddhartha<br><span>I’m a software developer</span></h2>
```

Comment:
- [ ] You have done great using the current font size here and correctly aligned text according to the Figma design. but it seems your color is a bit different from the Figma design. I'll suggest you get the correct color from Figma and update this. :+1: 
- - Your design:
![Screenshot from 2023-07-13 10-43-36](https://github.com/siddghosh108/Portfolio-Desktop-Version/assets/107632780/2a907c8f-cf6f-4b4e-985c-67cb86f5f104)

- - Figma design:
![Screenshot from 2023-07-13 10-43-51](https://github.com/siddghosh108/Portfolio-Desktop-Version/assets/107632780/4341fc92-9a42-4597-9992-7399b5aa8be6)

Reply:
![image](https://github.com/siddghosh108/Portfolio-Desktop-Version/assets/131663359/61c7b7fd-0492-479d-adfa-7f5eb40613bc)
I have changed the colour now